tests/zfs-tests/include/commands.cfg

```diff
@@ -182,6 +182,7 @@ export ZFS_FILES='zdb
     dbufstat
     zed
     zgenhostid
+    zstream
```

Comment:
To provide some test coverage for the new utility how about extending the existing `rsend/send-cD.ksh` and `cli_root/zfs_receive/zfs_receive_013_pos.ksh` to additionally use `zstream`.

Reply:
Thanks for pointing me to those tests.  I've updated them, please take a look and let me know if that's what you had in mind.